serverless-cljs-plugin/serverless_lumo/build.cljs

```diff
@@ -90,17 +96,20 @@
                      (write-index (.resolve path (:zip-path opts) "../index.js")))]
     (compile! (:source-paths cljs-lambda-opts) compiler)
     (zip! (:zip-path opts)
-          {:dirs  #{(output-dir compiler) "node_modules"}
-           :files #{[index {:name "index.js"}]}}
+          {:dirs  #{(output-dir compiler)}
+           :files #{[index {:name "index.js"}]}
+           :globs #{["node_modules/*" {:ignore (get-in opts [:package :exclude])}]}}
```

Comment:
This the only use case for now because we don't include anything else from the project. We could have `resource-paths` in the future for user-specific additions.